samples/dotnet/KafkaFunctionSample/SimpleKafkaTriggers.cs

```diff
@@ -7,50 +7,88 @@
 using System;
 using System.IO;
+using System.Text;
 
 namespace KafkaFunctionSample
 {
     public class SimpleKafkaTriggers
     {
-        [FunctionName(nameof(SampleConsumer))]
-        public void SampleConsumer(
-    [KafkaTrigger(
+        [FunctionName(nameof(ConsoleConsumer))]
+        public void ConsoleConsumer(
+        [KafkaTrigger(
             "LocalBroker",
-            "%EHTOPIC%",
+            "stringTopicTenPartitions",
             ConsumerGroup = "$Default",
-            Username = "$ConnectionString",
-            Password = "%EventHubConnectionString%",
-            Protocol = BrokerProtocol.SaslSsl,
-            AuthenticationMode = BrokerAuthenticationMode.Plain)] KafkaEventData<string> kafkaEvent,
-    ILogger logger)
+            AuthenticationMode = BrokerAuthenticationMode.Plain)] KafkaEventData<string>[] kafkaEvents,
+            ILogger logger)
         {
-            logger.LogInformation(kafkaEvent.Value.ToString());
+            foreach(var kafkaEvent in kafkaEvents)
+                logger.LogInformation(kafkaEvent.Value.ToString());
         }
 
-        [FunctionName(nameof(SampleProducer))]
-        public IActionResult SampleProducer(
+        [FunctionName(nameof(ConsoleProducer))]
+        public static IActionResult ConsoleProducer(
         [HttpTrigger(AuthorizationLevel.Function, "get", "post", Route = null)] HttpRequest req,
-        [Kafka(
-            "LocalBroker",
-            "%EHTOPIC%",
-            Username = "$ConnectionString",
-            Password = "%EventHubConnectionString%",
-            Protocol = BrokerProtocol.SaslSsl,
-            AuthenticationMode = BrokerAuthenticationMode.Plain)] out KafkaEventData<string>[] kafkaEventData,
-        ILogger logger)
+        [Kafka("LocalBroker", "stringTopicTenPartitions")] out string kafkaEventData,
+        ILogger log)
         {
-            var data = new StreamReader(req.Body).ReadToEnd();
-            kafkaEventData = new[] {
-                    new KafkaEventData<string>()
-                    {
-                        Value = data + ":1:" + DateTime.UtcNow.Ticks,
-                    },
-                    new KafkaEventData<string>()
-                    {
-                        Value = data + ":2:" + DateTime.UtcNow.Ticks,
-                    },
-                };
+            try
+            {
+                var data = new StreamReader(req.Body).ReadToEnd();
+                kafkaEventData = data + ":1:" + DateTime.UtcNow.Ticks;
+            }
+            catch (Exception ex)
+            {
+                throw new Exception("Are you sure the topic 'stringTopic' exists? To create using Confluent Docker quickstart run this command: 'docker-compose exec broker kafka-topics --create --zookeeper zookeeper:2181 --replication-factor 1 --partitions 10 --topic stringTopicTenPartitions'", ex);
+            }
+
             return new OkResult();
         }
+
+
+        // EventHubs Configuration sample
```

Comment:
nit: we should remove commented code

Reply:
Let me refactor this in the near future. We also have commented samples.  I create an issue for tracking this. https://github.com/Azure/azure-functions-kafka-extension/issues/188

Probably the best way is separate the project for each samples. We need to keep samples for the usage e.g. Avro, EventHubs, Confluent Cloud, and Local docker-compose for the customer convenience.